### ISSUE TITLE
Pesticid data

### ISSUE THREAD

**Issue by martincollignon** [martincollignon/landbrugsdata#15]
While we have pesticid consumption data, we don't have data about whether they are allowed, the allowed dosage, on which crops they're allowed to be spread on, obligatory distances from people and water and the potential dangers they can cause.

Potential data sources:

- web: middeldatabasen.dk
- web: https://mst.dk/erhverv/sikker-kemi/database-for-bekaempelsesmidler
- api: https://food.ec.europa.eu/plants/pesticides/eu-pesticides-database_en

Once should probably ask Miljøstyrelsen about it: mst@mst.dk

> Emne: Forespørgsel om offentligt tilgængelig oversigt over mindre anvendelser af pesticider
> 
> Jeg henvender mig for at høre, om Miljøstyrelsen har en frit tilgængelig og løbende opdateret oversigt over godkendelser til mindre anvendelse af pesticider (tidligere kaldet off-label godkendelser).
> 
> Specifikt er jeg interesseret i at vide:
> 
> Findes der en sådan oversigt på Miljøstyrelsens hjemmeside?
> - Hvis ja, hvor kan denne findes?
> - Hvor ofte opdateres en sådan oversigt?
> - Indeholder oversigten information om:
> -- Produktnavn
> -- Godkendelsesnummer
> -- Anvendelsesområde
> -- Dosering
> -- Godkendelsens varighed

**Comment by aleksanderbl29** [comment]
I will do this

**Comment by martincollignon** [comment]
did you reach out?

**Comment by aleksanderbl29** [comment]
Yes I did and they have responded this morning. I have pasted the contents below.

They referenced one of the databases you also linked above.

It looks like the database contains some of the information we would want. There is a name and registration number as well as a text body containing usage instruction/restrictions.

The database exports to excel but the download call could probably be put into a parser to ensure up-to-date info.

There is also a column containing a last-use date of the products which could be interesting to use to see if anyone uses products that are no longer allowed.

> Kære Aleksander
> 
> Tak for din e-mail.
> 
> Miljøstyrelsen har en database over godkendte pesticider: https://mst.dk/erhverv/sikker-kemi/database-for-bekaempelsesmidler/bmd som opdateres løbende når der sker ændringer. Du kan få en liste over mindre anvendelser ved at lave en søgning på alle lovlige og godkendte pesticider i menuen i højre side af skærmen, og derefter eksportere listen i excel. I kolonne Y fremgår, om der er godkendt mindre anvendelser til de enkelte produkter, tallene i kolonnen henviser til de enkelte mindre anvendelser (produktets registreringsnummer plus et nummer). 
> 
> For at se deltaljer om de enkelte mindre anvendelser kan brugsanvisningerne søges frem i bekæmpelsesmiddeldatabasen, ved at søge på produktnavnet i højre side af skærmen, derudover skal man klikke på "detaljer for alle produkter" øverst til venstre på skærmen, for at de mindre anvendelse fremgår af søgeresultatet.
> 
> Dvs. produktnavn og godkendelsesnummer søges frem i excelarket.
> Anvendelsesområde og dosering fremgår af brugsanvisningen for den mindre anvendelse
> Administrativt følger godkendelsen af den mindre anvendelse produktets godkendelse, dvs. de udløber samtidig, med mindre andet er angivet i brugsanvisningen.
> 
> 
> Du kan læse mere om mindre anvendelser på vores hjemmeside: https://mst.dk/erhverv/sikker-kemi/pesticider/godkendelse-af-pesticider/ansoegningsformer-og-krav/mindre-anvendelser

**Comment by martincollignon** [comment]
Nice. Would be interesting to see if there is a difference in data coverage
between this and middeldatabasen. I by the way tried doing OCR with most
LLMs on the etiquettes with quite poor results in september - back then
Gemini Flash (probably 1.5) was the best.

Martin Collignon
+45 42913110
Connect with me on Linkedin <http://hk.linkedin.com/in/martincollignon> or
Twitter <http://twitter.com/martincollignon>

**Comment by EdwardNgo** [comment]
Msg you on Upwork on this profile and my solution: https://www.upwork.com/freelancers/~015490ba7f66606aad.
